src/index.html

```diff
@@ -13,5 +13,166 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+          
+          <label class="form-field"> Surname:  
+            <input 
+              type="text" 
+              name="surname" 
+              autocomplete="off" 
+              required
+            >
+          </label><br/>
+          
+          <label class="form-field"> Name:
+            <input 
+              type="text" 
+              name="name" 
+              autocomplete="off" 
+              required
+            >
+          </label><br/>
```

Comment:
```suggestion
          </label>
          <br/>
```
Один тег - одна линия